docs/api/wanchain.md

```diff
@@ -71,9 +71,9 @@ A class instance `variable` holding a `GatewayConfig` which represents the confi
 
 | Argument | Description
 |-|-
-| assetLedgerDeployOrderId | A `string` representing an Ethereum address of the [asset ledger deploy gateway](/#public-addresses).
-| actionsOrderId | A `string` representing an Ethereum address of the [actions order gateway](/#public-addresses).
-| valueLedgerDeployOrderId | A `string` representing an Ethereum address of the [value ledger deploy gateway](/#public-addresses).
+| assetLedgerDeployOrderId | A `string` representing an Wanchain address of the [asset ledger deploy gateway](/#public-addresses).
+| actionsOrderId | A `string` representing an Wanchain address of the [actions order gateway](/#public-addresses).
```

Comment:
| actionsOrderId | A `string` representing a Wanchain address of the [actions order gateway](/#public-addresses).